src/core/jpg.js

```diff
@@ -28,6 +28,18 @@ let JpegError = (function JpegErrorClosure() {
   return JpegError;
 })();
 
+let DNLMarkerError = (function DNLMarkerErrorClosure() {
+  function DNLMarkerError(scanLines) {
```

Comment:
This is an atypical Error constructor. Where is `message`?

Reply:
(In this case, given that this `Error` is completely internal, it's perhaps not strictly needed.)

Fair point, I've updated the signature of `DNLMarkerError` to address this.